spec/requests/user_spec.rb

```diff
@@ -0,0 +1,45 @@
+require 'rails_helper'
+
+RSpec.describe 'Users', type: :request do
+  subject { User.new(name: 'Tom', photo: 'https://unsplash.com/photos/F_-0BxGuVvo', bio: 'Teacher from Mexico.') }
+  before { subject.save }
+
+  it 'is valid with valid attributes' do
+    expect(subject).to be_valid
+  end
+
+  it 'is not valid without a name' do
+    subject.name = nil
+    expect(subject).to_not be_valid
+  end
+
+  it 'is not valid without a photo' do
+    subject.photo = nil
+    expect(subject).to_not be_valid
+  end
+
+  it 'is not valid without a bio' do
+    subject.bio = nil
+    expect(subject).to_not be_valid
+  end
```

Comment:
- [ ] Please test the show and index actions using the right matchers as requested in the project.
>describe 'GET /index' do
  ...
end

>describe 'GET /show' do
  ...
end
![image](https://user-images.githubusercontent.com/91486791/223791267-3007530d-43c8-48bf-b41d-95f07fbf9ee4.png)